ng-pkgs/activation_service/package.py

```diff
@@ -0,0 +1,48 @@
+import os
+import sys
+
+import gevent
+
+from jumpscale.loader import j
+
+current_full_path = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(current_full_path + "/sals/")
+from activation_sal import create_gevent_pool
+
+
+class activation_service:
+    def install(self, **kwargs):
+        if "activation_wallet" not in j.clients.stellar.list_all():
+            secret = kwargs.get("secret", None)
+            network = kwargs.get("network", "TEST")
+            wallet = j.clients.stellar.new("activation_wallet", secret=secret, network=network)
+            if not secret:
+                if network == "TEST":
+                    wallet.activate_through_friendbot()
+                else:
+                    wallet.activate_through_threefold_service()
+            wallet.save()
+
+        location_actors_443 = j.sals.nginx.main.websites.default_443.locations.get(name="activation_actors")
+        location_actors_443.is_auth = False
```

Comment:
is this still required if it is already set in the package.toml?

Reply:
This is to make the actor accessible but it is not added in the package.toml, the rewrite in the package.toml is used to rewrite `/activation_service/` to the endpoint of the actor which is accessible but much longer `/activation_service/actors/activation_service/`